server/routes/user.js

```diff
@@ -0,0 +1,27 @@
+/* eslint new-cap: "off" */
+const express = require('express');
+
+const router = express.Router();
+const User = require('../controllers/user');
+const Authentic = require('../controllers/middleware');
+
+router.route('/users/login')
+  .post(User.login);
+
+router.route('/users')
+  .post(User.create);
+
+router.use(Authentic.Auth);
+
+router.route('/users')
+  .get(User.all);
+
+router.route('/users/:id')
+  .get(User.get)
+  .delete(User.delete)
+  .put(User.update);
+
+router.route('/users/logout')
```

Comment:
They don't cause a conflict because the`/users/:id` route doesn't have a post method and the `/users/logout` route is a post operation. This means that they are completely different routes and don't match.